[README] Clarify DShot support on MAIN outputs for OrangeCubePlus

Mystraht opened the pull request:
Tested on Orange Cube Plus today, confirming that DShot works as expected when BRD_IO_DSHOT is set to 1 which is DShot fw

rmackay9 commented:
Hi @Mystraht,

Thanks for this.  Could you squash the two commits together and prefix the commit title with "AP_HAL_ChibiOS: "?  To be clear I mean the commit title, not the PR title.

Mystraht commented:
> Hi @Mystraht,
> 
> Thanks for this. Could you squash the two commits together and prefix the commit title with "AP_HAL_ChibiOS: "? To be clear I mean the commit title, not the PR title.

Hi,

Thanks for the clarification, I've squashed the commits and updated the commit title as requested.

peterbarker commented:
Merged thanks!

rmackay9 commented:
FYI @bugobliterator
